AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/DataStoreListTests.swift

```diff
@@ -27,13 +28,13 @@ class ListTests: BaseDataStoreTests {
         Amplify.DataStore.query(Post.self, byId: postId) {
             switch $0 {
             case .success(let result):
-                if let post = result {
-                    if let comments = post.comments {
+                if let post = result, let postComments = post.comments {
+                    if let comments = postComments as? DataStoreList<Comment> {
                         XCTAssert(comments.state == .pending)
```

Comment:
post.comments is a List and `state` is/continues to be internal that was moved from List to DataStoreList, hence the casting before ensuring `state` is accurate